test/form/index.js

```diff
@@ -37,6 +38,11 @@ describe('form', () => {
 			const createBundle = () => promise || (promise = rollup.rollup(options));
 
 			FORMATS.forEach(format => {
+				const skipBecauseNoSystem = format === 'system' && !fs.existsSync(path.resolve(__dirname, samples, dir, '_expected', 'system.js'))
+				if (skipBecauseNoSystem) {
+					return;
```

Comment:
Don't call `it()` for "form" tests that don't have a system format expectation.

This prevents breaking existing tests but also providing a consistent way to add systemjs expectations in future tests.

Reply:
This can also be avoided by rebasing to the other branch.